examples/event/main.go

```diff
@@ -0,0 +1,73 @@
+package main
```

Comment:
This file is great to know how event package is used, but we don't need this file in Ebiten.

Reply:
OK, I will remove this.